### DISCUSSION TITLE
breaks configuration file

### DISCUSSION THREAD

**Discussion by eloquentize** [spatie/laravel-markdown#74] [General]
Hi !
On laravel 11

```
'extensions' => [
        new \League\CommonMark\Extension\Table\TableExtension(),
],
```

breaks configuration and result to "Your configuration files are not serializable." error on php artisan optimize
bootstrap/cache/config.php:720
Error::("Call to undefined method League\CommonMark\Extension\Table\TableExtension::__set_state()")

**Comment by jtouza** [comment]
do this:

   'extensions' => [
        \League\CommonMark\Extension\Table\TableExtension::class,
    ],
